test/land/test_imsproc.sh

```diff
@@ -53,6 +53,12 @@ done
 ulimit -Ss unlimited
 ${EXECDIR}/calcfIMS.exe
 
+export PYTHONPATH=$PYTHONPATH:${project_source_dir}/build/lib/python3.7/pyioda/
```

Comment:
the `python3.7` part here is dangerous. @guillaumevernieres did you ever figure out how to do this the correct way?

Reply:
Sorry, just seeing this. I don't think this is the correct way, but probably safer:
[stage.py#L19](https://github.com/NOAA-EMC/GDASApp/blob/9143acb3a1f5043e09b3ef7f67e72d536d50eb23/ush/ufsda/stage.py#L19)

Reply:
@guillaumevernieres Thanks for your reply. Do you suggest me to use something like `'python3.'+str(sys.version_info[1])` to replace `python3.7`?

Reply:
@guillaumevernieres Or to modify `imsfv3_scf2ioda.py` following the approach in the `stage.py`.

Reply:
There must be some CMAKE variable here to use? How does CMAKE know to create/put the IODA python in this spot in the first place?

Reply:
`python${Python3_VERSION_MAJOR}.${Python3_VERSION_MINOR}`
See: https://github.com/JCSDA-internal/ioda/blob/d9250472620f039dd8c590acee0350417206fc94/src/engines/ioda/python/CMakeLists.txt#L9

Reply:
@CoryMartin-NOAA Because I have little experience in the cmake usage, I still don't have an idea how to proceed and need further help with this. 

From my limited investigation, it seems both `Python3_VERSION_MAJOR` and `Python3_VERSION_MINOR` should be generated during the IODA build process.

Reply:
@jiaruidong2017 does it work to do:
```
export PYTHONPATH=$PYTHONPATH:${project_source_dir}/build/lib/python${Python3_VERSION_MAJOR}.${Python3_VERSION_MINOR}/pyioda/
```
?

Reply:
No, because both `Python3_VERSION_MAJOR` and `Python3_VERSION_MINOR` are empty.

Reply:
See the `PYTHONPATH` below: 
/scratch2/NCEPDEV/stmp1/Jiarui.Dong/gdasapp/gdasapp.4/build/lib/`python.`/pyioda/

Reply:
ah, okay, so we need to figure out how to get them set... ok let me think about it, stay tuned.

Reply:
@CoryMartin-NOAA Thanks.

Reply:
Oh, this is a shell script, did you try setting them it in the CMake first?

Reply:
yes

Reply:
@CoryMartin-NOAA I don't know how to set both `Python3_VERSION_MAJOR` and `Python3_VERSION_MINOR` in the CMakeLists.txt correctly. Do you have any example for me to learn how to set them? Thank you very much.

Reply:
https://github.com/NOAA-EMC/GDASApp/blob/98a8294aa8f89da2e40699d833389079fccd370e/test/land/CMakeLists.txt#L15
Change to something like:
```
"GDASAPP_TESTDATA=$ENV{GDASAPP_TESTDATA};Python3_MAJOR_VERSION=${Python3_MAJOR_VERSION};Python3_MINOR_VERSION=${Python3_MINOR_VERSION}"
```
I think that should do the trick

Reply:
Thanks @CoryMartin-NOAA for your great help. Is it `Python3_MAJOR_VERSION` or `Python3_VERSION_MAJOR`?

Reply:
Should be what I wrote before, ${Python3_VERSION_MAJOR}, not what I wrote just now from memory :-)

Reply:
@CoryMartin-NOAA It worked perfectly. I will push the changes to the repo. Thanks again.